ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -37,20 +37,21 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
   /**
    * Loads all targets under current scope.
```

Comment:
Can this comment be updated to reflect the new returns?

Reply:
I already had updated the `@return`, are you referring to something else?

Reply:
`Loads all targets under current scope` should we mention the total items being returned here?